.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,12 @@
+# Grocery Store
+Congratulations! You're submitting your assignment.
+
+## Comprehension Questions
+
+|  Question 	|   Response	|
+|---	|---	|
+|   Why is it useful to put classes inside modules?  What do you think?	|   	|
+|   What is accomplished with `raise ArgumentError`?  What do you think it's doing?	|   	|
```

Comment:
Do we really need the `What do you think?` or `What do you think it's doing?` here? Seems a little pedantic.